client/components/telecom/telephony/group/number/feature/ovhPabx/menu/entry/edit/telephony-group-number-feature-ovh-pabx-menu-entry-edit.html

```diff
@@ -392,17 +391,17 @@ <h2 data-translate="telephony_number_feature_ovh_pabx_menu_entry_edit_title"></h
                 <div class="section-group">
                     <!-- ADD NEW MENU -->
                     <button type="button"
-                            class="section-item no-style full-width text-left"
+                            class="section-item no-style w-100 text-left"
```

Comment:
-`no-style` (`btn btn-link` maybe?).

Reply:
I only have two hands, I cannot fix all the problems in the manager with this pull request. I will focus my attention only on lines that I have modified. Theres more than 100 button in this situation, which is real sad. This should be rectify in a future branch...